regulations/templates/regulations/chrome-preamble-search.html

```diff
@@ -0,0 +1,34 @@
+{% extends "regulations/preamble-base.html" %}
+
+{% block title %}
+Search of {{reg_part}} for {{q}} | eRegulations
+{% endblock %}
+
+{% block active_title %}
+<em class="header-label">
+<span class="subpart">Search results</span> {{ results.total_hits }} result{{ results.total_hits|pluralize }}
+</em>
+{% endblock %}
+
+{% block chrome-content %}
+<main role="main">
+  <div id="content-body" class="main-content">
+      {% load render_nested %}
+      {{ main_content|safe }}
```

Comment:
Looks like this is safe to drop here, but not in the base template, since `main_content` is still set there. I'll make the change here, and we can purge the last `main_content` separately.